ci/linux-debian.Dockerfile

```diff
@@ -28,12 +28,11 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
         git ca-certificates wine64 wine python3-simplejson python3-six msitools winbind procps && \
     git clone https://github.com/mstorsjo/msvc-wine && \
     mkdir /opt/msvc && \
-    python3 msvc-wine/vsdownload.py --accept-license --dest /opt/msvc Microsoft.VisualStudio.Workload.VCTools && \
-    msvc-wine/install.sh /opt/msvc
+    python3 msvc-wine/vsdownload.py --accept-license --dest /opt/msvc Microsoft.VisualStudio.Workload.VCTools
 
-# Moving the wine prefix to /tmp avoids error D8037 when invoking cl.exe.
-ENV WINEPREFIX=/tmp/wineprefix
-# Initialize the wine prefix. Wait until the wineserver process has
+# Since commit 2146cbfaf037e21de56c7157ec40bb6372860f51, the
+# msvc-wine effectively initializes the wine prefix when running
+# the install.sh script. Wait until the wineserver process has
 # exited before closing the session, to avoid corrupting the wine prefix.
-RUN wine64 wineboot --init && \
+RUN msvc-wine/install.sh /opt/msvc && \
     while (ps -A | grep wineserver) > /dev/null; do sleep 1; done
```

Comment:
Nit: maybe this should be merged with the `RUN` command above.

(but let's first see if it works at all :) nice catch!)

Reply:
> edit: This is my suggestion

Thanks! [Updated](https://github.com/bitcoin-core/secp256k1/pull/1337#issuecomment-1574769542).